scripts/ci.sh

```diff
@@ -80,7 +82,9 @@ push() {
 		make -j$NPROC latest
 		docker system prune -a -f
 		install_qemu
-		make latest_arm
+		GOARCH=arm GOARM=7 make latest
```

Comment:
Please paralelise the compilation.

Reply:
Tnx, it was removed in debuging process and then I put it back in wrong `make` command